mlflow/server/__init__.py

```diff
@@ -281,6 +283,8 @@ def _run_server(  # noqa: D417
     if expose_prometheus:
         env_map[PROMETHEUS_EXPORTER_ENV_VAR] = expose_prometheus
 
+    env_map[FLASK_SERVER_SECRET_KEY] = str(uuid.uuid4())
```

Comment:
there should be no effect on the regular app, since this key is only read in the auth app